models/StandaloneFundingPlanParameters.v1.json

```diff
@@ -13,10 +13,8 @@
           "enum": [
             "FUNDING_PLAN_PARAMETERS"
           ],
-          "description": "Type du jeu de paramètres."
-        },
-        "parameters": {
-          "$ref": "./FundingPlanParameters.v1.json"
+          "description": "Type du jeu de paramètres.",
+          "default": "FUNDING_PLAN_PARAMETERS"
```

Comment:
We can't avoid the default value, equal to the only value available in the enum hehe ?

Reply:
I guess we could, I'm not so clear on what it means regarding libraries that could auto-fill values or not if not present.
In our server (the typescript one), if a value is missing we fill it the default value automatically, I didn't test extensively though so I'm not 100% confident on the pros and cons of this default